cmd/kubeapps-apis/README.md

```diff
@@ -16,6 +16,17 @@ Together, this enables the best of both worlds: a well known and used Interface
 
 Finally, we've also chosen to use the [buf](https://buf.build/) tool for generating the code from the proto files. In the past we've used `protoc` (proto buffer compiler) and its extensions directly, but `buf` allows you to specify a simple yaml config instead, and also provides a `lint` command to ensure that your choice of API structure follows best practise, as well as ensuring you're aware when you break backwards compatability.
 
+## Plug-able
+
+The kubeapps-apis service uses the standard [go plugin package](https://golang.org/pkg/plugin/) to be able to load api plugins at run-time.
+
+Each plugin consists of 2 source files (and some generated files):
+
+* A `.proto` file defining the service that uses the messages defined in relevant part of kubeappsapis.core,
+* A `main.go` that compiles to an .so file for that plugin. This `main.go` has two public functions: one to register the plugin with a GRPC server and one to register the plugin for the http handler as well as the implementation for the server. This may be split into further modules as the complexity of the plugin grows.
```

Comment:
Perhaps obvious, but I'd say the `.go` files are is under `./plugins/<pluginName>` whereas the `.proto` are under `./proto/kubeappsapis/plugins/<pluginName>`

Reply:
Done, thanks.